backend/lab-service/src/main/java/org/mltooling/lab/endpoints/LabAdminEndpoint.java

```diff
@@ -161,7 +163,7 @@ public Response resetAllWorkspaces(
   @ApiOperation(
       value =
           "Resets a workspace. Removes the container (keeps all persisted data) and starts a new"
-              + " one.",
+              + " one.  When specified an image, it is used this image instead",
```

Comment:
Maybe reformulate for clarity: "If an image is specified, it will be used instead of the default image"

Reply:
Changed in [781b1d0](https://github.com/SAP/machine-learning-lab/pull/26/commits/781b1d017d0f46a8bc3a18289b453e10309ebaa2)